backup/v1.2.1/utils/db_manager.py

```
@@ -0,0 +1,322 @@
+        rows = cursor.fetchall()
+        
+        result = []
+        for row in rows:
+            download = dict(row)
+            
+            # Parse metadata from JSON
+            if 'metadata' in download and download['metadata']:
+                try:
+                    metadata = json.loads(download['metadata'])
+                    download.update(metadata)
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)